charts/pega/templates/_pega-credentials-secret.tpl

```diff
@@ -15,6 +15,12 @@ data:
   CASSANDRA_USERNAME: {{ .Values.dds.username | b64enc }}
   # Base64 encdoded password for connecting to cassandra
   CASSANDRA_PASSWORD: {{ .Values.dds.password | b64enc }}
+  {{ if $.Values.hazelcast.enabled }}
+  # Base64 encoded username used for authentication in hazelcast client server mode
+  HZ_CS_AUTH_USERNAME: {{ .Values.hazelcast.username | b64enc }}
+  # Base64 encoded password used for authentication in hazelcast client server mode
+  HZ_CS_AUTH_PASSWORD: {{ .Values.hazelcast.password | b64enc }}
+  {{ end }}
```

Comment:
Does Pega node require these credentials? If not its worth considering creating separate secret for hazlecast credentials